tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -531,17 +531,29 @@ def _execute_test_suite(self, suite: TestSuiteInfo) -> bool:
                                 TestStatus.FAILED,
                                 test_start_time,
                                 message=str(e))
+                        except RemoteTestError as e:
+                            test_success = False
+                            summary.append(f"[Failed]  {test.name}")
+                            message = f"UNEXPECTED ERROR IN [{e.command}] {e.stderr}\n{e.stdout}"
+                            log.error("******** [Failed] %s: %s", test.name, message)
+                            self.context.lisa_log.error("******** [Failed] %s", test_full_name)
+                            self._report_test_result(
+                                suite_full_name,
+                                test.name,
+                                TestStatus.FAILED,
+                                test_start_time,
+                                message=str(message))
                         except:  # pylint: disable=bare-except
                             test_success = False
                             summary.append(f"[Error]   {test.name}")
-                            log.exception("UNHANDLED EXCEPTION IN %s", test.name)
-                            self.context.lisa_log.exception("UNHANDLED EXCEPTION IN %s", test_full_name)
+                            log.exception("UNEXPECTED ERROR IN %s", test.name)
```

Comment:
Changed this from "UNHANDLED EXCEPTION" to "UNEXPECTED ERROR" since the former gives the impression that the tests should handle those exceptions, but tests can actually just let those exceptions bubble up when doing test setup, etc.

Note that some unexpected errors should actually be handled by the tests, though